Makefile

```diff
@@ -55,7 +55,7 @@ $(WEAVER_EXE) $(WEAVEDNS_EXE) $(WEAVEPROXY_EXE) $(NETCHECK_EXE): common/*.go com
 		false; \
 	}
 
-$(WEAVER_EXE): router/*.go ipam/*.go ipam/*/*.go prog/weaver/main.go
+$(WEAVER_EXE): router/*.go ipam/*.go ipam/*/*.go gossipdns/*.go prog/weaver/main.go
```

Comment:
We have discussed in the past that we should move things to subdirectories, specially when they are not shared between different components... I think this would be a good candidate for such kind of change...